### PR TITLE
Bug 1911809: [interop-ui-tests] update steps in install helm chart feature file

### PR DESCRIPTION
**Pull request by shvasude**
Jira Id: https://issues.redhat.com/browse/HELM-15, https://issues.redhat.com/browse/HELM-66
Bugzill ID: Bug [1911809](https://bugzilla.redhat.com/show_bug.cgi?id=1911809)
Update steps for scenario:
* Install Helm Chart from Developer Catalog Page using YAML View: HR-03
* Install Helm Chart from +Add Page using Form View: HR-02-TC01
* Chart versions drop down menu

### CODE REVIEW

File: frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature [hunk 1]
```diff
@@ -1,84 +1,84 @@
 Feature: Install the Helm Release
     As a user, I want to install the helm release
 
-Background:
-    Given user is at developer perspecitve
-    And user has selected namespace "aut-helm-installation"
+    Background:
+        Given user is at developer perspecitve
```
Comment (makambalaji) [inline frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature:5]:
```suggestion
        Given user is at developer perspective
```

Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature:5]: https://github.com/openshift/console/pull/7656/files address the same

Comment (shvasude) [reply frontend/packages/dev-console/integration-tests/features/helm/install-HelmChart.feature]: Done, https://github.com/openshift/console/pull/7680/commits/9c701865e803bf81296ef5e1eb0f9cf38187dd6d#

**Comment by makambalaji** [comment]
Please add bugzilla number for this pr

**Comment by shvasude** [comment]
> Please add bugzilla number for this pr

Added the JIRA ID

**Pull request merged**